modules/01 JavaScript Programming/Assignments/8/answer8

```diff
@@ -0,0 +1,6 @@
+var bloodPressure=70;
+var isNormal=false;
+isNormal = bloodPressure>=80&&bloodPressure<=120;
```

Comment:
you should surround the AND with spaces:
bloodPressure>=80 && bloodPressure<=120;